picard/file.py

```diff
@@ -541,6 +546,21 @@ def is_saved(self):
         return self.similarity == 1.0 and self.state == File.NORMAL
 
     def update(self, signal=True):
+        now = time()
```

Comment:
Couldn't we use the `@throttle(250)` decorator from `picard.util` for this?

Reply:
Nope, i tried, but properties are changing, so it doesn't work (i don't remember exactly what was the issue, but i ended with this solution, because it didn't work).
If you can, do a test with a lot of files on windows, i'm not even sure it is worth.